scripts/dumpapp

```diff
@@ -24,28 +22,17 @@ def main():
   # transport.
   device = os.environ.get('ANDROID_SERIAL')
   try:
-    conn = HTTPConnectionOverADB(device, process)
-    query_params = ['argv=' + urllib.parse.quote(arg) for arg in args]
-    url = '/dumpapp?%s' % ('&'.join(query_params))
-    fake_host = { 'Host': 'localhost:5037' }
+    sock = stetho_open(device, process)
 
-    http_method = None
-    body = None
-    if not sys.stdin.isatty():
-      http_method = 'POST'
-      body = sys.stdin.detach().read()
-    else:
-      http_method = 'GET'
-      body = None
-
-    conn.request(http_method, url, body, fake_host)
-
-    reply = conn.getresponse()
+    enter_frame = b'!' + struct.pack('!L', len(args))
```

Comment:
Ideas to consider here:
1) Make the magic more than one character.
2) Include a version number to make reporting version mismatches simpler in the future.
3) As discussed, this might need to look sufficiently like http that if this code connects to an old Stetho server we can deterministically detect and report the version mismatch error.